shell/common/platform_view.h

```diff
@@ -51,13 +51,17 @@ namespace flutter {
 ///
 class PlatformView {
  public:
+  using AddViewCallback = std::function<void(bool added)>;
+  using RemoveViewCallback = std::function<void(bool removed)>;
   //----------------------------------------------------------------------------
   /// @brief      Used to forward events from the platform view to interested
   ///             subsystems. This forwarding is done by the shell which sets
   ///             itself up as the delegate of the platform view.
   ///
   class Delegate {
    public:
+    using AddViewCallback = PlatformView::AddViewCallback;
```

Comment:
Nit: Is this necessary? You can just use the typedef from the parent.

Reply:
The compiler requires `Shell` to use `PlatformView::AddViewCallback` and does not accept simply `AddViewCallback` without it. It's definitely not necessary, and I'm down to remove it.